hack/update-codegen.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# more info: https://cloud.redhat.com/blog/kubernetes-deep-dive-code-generation-customresources
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+mkdir -p ./api/clustergroupupgradesoperator
+cd ./api/clustergroupupgradesoperator
+ln -s ../v1alpha1 ./v1alpha1
+cd ../../
+
+SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+CODEGEN_PKG=${CODEGEN_PKG:-$(cd "${SCRIPT_ROOT}"; ls -d -1 ./vendor/k8s.io/code-generator 2>/dev/null || echo ../code-generator)}
+
+# TIP: append '-v 10' with the script call below for logs
+bash ${CODEGEN_PKG}/generate-groups.sh "all" \
+    "github.com/openshift-kni/cluster-group-upgrades-operator/pkg/generated" \
+    "github.com/openshift-kni/cluster-group-upgrades-operator/api" \
+    clustergroupupgradesoperator:v1alpha1 \
+    --output-base "./pkg" \
+    --go-header-file hack/boilerplate.go.txt
+
+cp -rp "./pkg/github.com/openshift-kni/cluster-group-upgrades-operator/pkg/" "./pkg/"
+
+sed -i '' "s|api/clustergroupupgradesoperator/v1alpha1|api/v1alpha1|g" $(find pkg/generated -type f)
```

Comment:
The '' is causing your regex to be seen as a file argument, causing the following error:
sed: can't read s|api/clustergroupupgradesoperator/v1alpha1|api/v1alpha1|g: No such file or directory

In turn, this causes update-codegen.sh to fail with exit code 2.

Reply:
Sorry about that. Ya needed to add '' for macOS cause [apparently](https://stackoverflow.com/questions/16745988/sed-command-with-i-option-in-place-editing-works-fine-on-ubuntu-but-not-mac) sed works differently there. Added better support. Can you give it shot again?